internal/core/core_test.go

```diff
@@ -116,6 +117,24 @@ func TestMetadata(t *testing.T) {
 	require.Equal(t, meta, expected)
 }
 
+func TestCustomAxisOrder(t *testing.T) {
+	expected := []*Axis{
+		{Annotation: "Inline", Min: 1, Max: 5, Samples: 3, StepSize: 2, Unit: "unitless"},
+		{Annotation: "Crossline", Min: 10, Max: 11, Samples: 2, StepSize: 1, Unit: "unitless"},
+		{Annotation: "Sample", Min: 4, Max: 16, Samples: 4, StepSize: 4, Unit: "ms"},
+	}
+	handle, _ := NewVDSHandle(well_known_custom_order)
+	defer handle.Close()
+	buf, err := handle.GetMetadata()
+	require.NoErrorf(t, err, "Failed to retrieve metadata, err %v", err)
```

Comment:
Note that error is already printed by framework, so you can drop it, here and below.

Reply:
Sure